oak-lucene/pom.xml

```diff
@@ -220,9 +219,9 @@
 
     <!-- General utility libraries -->
     <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <scope>provided</scope>
+      <groupId>org.apache.jackrabbit</groupId>
```

Comment:
...already present above due to the change for oak-search

Reply:
Removed